docs/rest-apis/event-gateway/schemas.md

```
@@ -0,0 +1,1518 @@
+      },
+      {
+        "name": "pull_requests",
+        "method": "SUB"
+      }
+    ]
+  }
+}
+
+```
+
+### Properties
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)